dev-libs/sdl_kitchensink/sdl_kitchensink-1.0.9.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="FFmpeg and SDL2 based library for audio and video playback"
+DESCRIPTION="SDL_kitchensink is a FFmpeg and SDL2 based library for audio and video playback, written in C99."
+HOMEPAGE="https://github.com/katajakasa/SDL_kitchensink"
+COPYRIGHT="Tuomas Virtanen"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/katajakasa/SDL_kitchensink/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="cb92976adc75523c6c1ee086e2e1f1ffd9df99b7d51b0667b68164f2063adf09"
+SOURCE_FILENAME="SDL_kitchensink-$portVersion.tar.gz"
+SOURCE_DIR="SDL_kitchensink-1.0.9"
+
+ARCHITECTURES="all"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	sdl_kitchensink$secondaryArchSuffix = $portVersion
+	lib:libSDL_kitchensink$secondaryArchSuffix = $portVersion
```

Comment:
You can use libVersionCompat instead of portVersion here, look at some other recipes on how they do it :)